benchmark/single-source/SetTests.swift

```diff
@@ -252,6 +497,37 @@ public let SetTests = [
     tags: [.validation, .api, .Set],
     setUpFunction: { blackHole([setP, setQ]) }),
 
+  BenchmarkInfo(
+    name: "Set.Intersection.Sequence.Int0",
```

Comment:
Lowercase `i`, intersection is a method name.
```suggestion
    name: "Set.intersection.Seq.Int0",
```
… also in all `intersection` benchmarks below.